### ISSUE TITLE
batch norm parameters in the pretrained imagenet resnet

### ISSUE THREAD

**Issue by zacwellmer** [tensorflow/models#7045]
### System information
- **What is the top-level directory of the model you are using**: models/official/resnet
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: N/A
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: N/A
- **TensorFlow installed from (source or binary)**: N/A
- **TensorFlow version (use command below)**: 1.13.1
- **Bazel version (if compiling from source)**: N/A
- **CUDA/cuDNN version**: N/A
- **GPU model and memory**: N/A
- **Exact command to reproduce**: N/A

### Describe the problem
the provided [pretrained resnet on imagenet](http://download.tensorflow.org/models/official/20181001_resnet/savedmodels/resnet_v2_fp32_savedmodel_NHWC.tar.gz) doesn't have seem to apply conv batch norm in the same fashion as the the current implementation.

if you download the above SavedModel and look at the variables you will notice that some conv layers(4 in total) show up w/out being followed by batch norm. In the current implementation all the conv layers are proceeded by batch norm.
```
import tensorflow as tf
export_dir='resnet_v2_fp32_savedmodel_NHWC/1538687283/'
with tf.Session(graph=tf.Graph()) as sess:
    tf.saved_model.loader.load(sess, ['serve'], export_dir)
    print([ v for v in tf.global_variables()])
```

**Comment by zacwellmer** [comment]
ahhh my bad, I did not realize the default resnet version is 1
